### ISSUE TITLE
Mevalonate aciduria

### ISSUE THREAD

**Issue by jpazmandi** [obophenotype/human-phenotype-ontology#6116]
**Preferred term label:**
Mevalonate aciduria

**Synonyms**
mevalonate kinase deficiency

**Definition (free text, please give PubMed ID)**
Severe form of mevalonate kinase deficiency

**Parent term (use hpo.jax.org/app)**
HP:0012072

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
610377

**Your nano-attribution (ORCID)**
0000-0003-3411-9598

**Comment by pnrobinson** [comment]
I will add this as a synonym to this existing term.
https://hpo.jax.org/app/browse/term/HP:0032638
